src/components/ArticleCard.jsx

```diff
@@ -1,21 +1,19 @@
-
+import { Link } from "react-router-dom";
+import Card from 'react-bootstrap/Card';
 
 function ArticleCard ({article}) {
-    return (<article>
-                <img src={article.article_img_url} alt={article.title} />
-                    <h2>{article.title} </h2>
-                <ul>
-                    <li>
-                        Author: {article.author}
-                    </li>
-                    <li>
-                        Topic: {article.topic}
-                    </li>
-                    <li>
-                        Votes: {article.votes}
-                    </li>
-                </ul>
-            </ article>
+    return (
+        <Card className="article">
+            <Link to={`articles/${article.article_id}`} >
```

Comment:
Consider allowing users to click on any part of the card in order to navigate to the single article for ease of use